extensions/amp-wordpress-embed/validator-amp-wordpress-embed.protoascii

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-wordpress-embed
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-wordpress-embed"
+    version: "1.0"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-wordpress-embed>
+  html_format: AMP
+  tag_name: "WORDPRESS-EMBED"
+  requires_extension: "amp-wordpress-embed"
+  attr_lists: "extended-amp-global"
+  spec_url: "https://amp.dev/documentation/components/amp-wordpress-embed/"
+  attrs: {
+    name: "data-url"
+    value_url: {
+      protocol: "https"
+      allow_relative: false
+    }
+    mandatory: true
+  }
+  amp_layout: {
+    supported_layouts: CONTAINER
```

Comment:
Why is `container` layout supported? does the component take children?

Reply:
Removed the layout.